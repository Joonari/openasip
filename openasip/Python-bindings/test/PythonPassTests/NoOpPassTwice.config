<scheduler-configuration>

	<pass>
		<module>
			<name>NoOpPass1</name>
			<file>NoOpPass.py</file>
		</module>
	</pass>
	<pass>
		<module>
			<name>NoOpPass2</name>
			<file>NoOpPass.py</file>
		</module>
	</pass>

</scheduler-configuration>
