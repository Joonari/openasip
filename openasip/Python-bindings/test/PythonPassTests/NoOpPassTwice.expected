NoOpPass instantiated
needsTarget() called
needsProgram() called
needsProgramRepresentation() called
NoOpPass instantiated
needsTarget() called
needsProgram() called
needsProgramRepresentation() called
setProgram() called with <TCE.base.program.Program object at 0x........>
setTarget() called with <TCE.base.mach.Machine object at 0x........>
Running NoOpPass.py
setProgram() called with <TCE.base.program.Program object at 0x........>
setTarget() called with <TCE.base.mach.Machine object at 0x........>
Running NoOpPass.py
